### PR TITLE
SEO improvements (content wider than screen) and other smaller fixes

### PR DESCRIPTION
**Pull request by msbt**
## Summary of the changes / Why this is an improvement

Draft because there's more to come

Here are again some smaller fixes for better usability, to avoid content to be larger than the screen:

### Tables (on <991px width they get a horizontal scrollbar - waiting for more infos und how the tables work):
grafik

### Fieldlists columns (on <560px width they become rows

Before:
grafik

After:
grafik

### SQL-99 (on <991px the whitespace at the top gets removed)

Before:
grafik

After:
grafik

### CODE REVIEW

File: src/crate/theme/rtd/conf/__init__.py [hunk 1]
```diff
@@ -113,7 +113,7 @@
 
 # Configure OpenGraph extension
 ogp_site_url = "https://crate.io/"
-ogp_description_length = 300
+ogp_description_length = 150
```
Comment (amotl) [inline src/crate/theme/rtd/conf/__init__.py:116]: Just for the records. Can you elaborate again why this change was made?

**Review by amotl — APPROVED** [review]
Thank you very much. I just added a small question about this detail:

> SEO: Change opengraph description length from 300 to 150 characters

File: src/crate/theme/rtd/conf/__init__.py (hunk 1 above)
Comment (msbt) [reply src/crate/theme/rtd/conf/__init__.py:116]: @amotl Even though there's no real length limit (there might be one at 10k chars), 150 - 160 characters is the most popular recommendation for the og description (which is also the recommended length for the regular meta description).

Like https://www.opengraph.xyz/ shows popular services and the corresponding description

**Review by msbt — COMMENTED** [review]
sorry, wrong tab 😬
